### PR TITLE
Complete : Restaurant App Page (Mastering webpack)

### PR DESCRIPTION
**Pull request by imahnama**
## In this project I:
- Install Wepback cli
- Create bare Html Homepage using Javascript
- Create Menu Page and Contact page using Javascripts
- Style pages using css
- Add linter files
- Add readme file.

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      </div>
+      <div class="form-group">
+        <label for="message" class="message">Message:</label>
+        <textarea name="message" id="" cols="50" rows="7"></textarea>
+      </div>
+      <div class="form-group">
+      <button class="btn btn-primary ">
+            Submit
+          </button>
+      </div>
```
Comment (bellom) [inline src/contact.js]: We need to use `document.createElement` and `document.appendChild` to appends HTML elements to the `<div>` with `id=content`

image

https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  const menu = document.createElement('div');
+  menu.setAttribute('id', 'main-menu');
+
+  mainMenuDiv.appendChild(menu);
+
+  // const menuHeader = document.createElement('h2');
+  // menuHeader.innerHTML = 'Our Menu';
+  //
+  // menu.appendChild(menuHeader);
```
Comment (bellom) [inline src/menu.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: src/menu.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  divFive.appendChild(img5);
+  divFive.appendChild(menuParagraphFive);
+  menuItems1.appendChild(divSix);
+  divSix.appendChild(img6);
+  divSix.appendChild(menuParagraphSix);
+  // menuItems.appendChild(img1);
+  // menuItems.appendChild(img2);
+  // menuItems.appendChild(img3);
+  // menuItems.appendChild(img4);
+  // menuItems.appendChild(img5);
```
Comment (bellom) [inline src/menu.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @imahnama  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/menu.js (hunk 2 above)
Comment (imahnama) [reply src/menu.js]: noted

File: src/menu.js (hunk 3 above)
Comment (imahnama) [reply src/menu.js]: noted

**Comment by imahnama** [comment]
> ## Hi @imahnama 👏
> ### Awesome job so far!! 👍
> #### You have done wonderfully well, you passed `almost` all the checklist for this project 👍 just a few issues left to work on............ 🙂
> **Kindly check out my comments below;**
> 
> ### ♻️ Kindly re-submit for review after making all those changes 🙂 Thumps -up await you.....
> **Best,**
> 
> **Oluwaseun Bello**
> **_Slack Handle - @sean bellom_**

Thank you very much for your reviews @bellom . I have made implemented the changes.

**Review by bellom — APPROVED** [review]
## Hi @imahnama :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**